extension/src/experiments/columns/collect/index.test.ts

```diff
@@ -821,3 +720,11 @@ describe('collectChanges', () => {
     ])
   })
 })
+
+describe('collectRelativeMetricsFiles', () => {
```

Comment:
[F] This was moved from `plots/data/collect` to encapsulate the data type inside of `Experiments`.